### PR TITLE
A little README update

### PR DESCRIPTION
**Pull request by MolassesLover**
- Cleaned up build instructions and made them consistent with my updated [`tasks.json`](https://github.com/turanszkij/WickedEngine/pull/462/files#diff-a5de3e5871ffcc383a2294845bd3df25d3eeff6c29ad46e3a396577c413bf357).
- Added the Wicked Engine Demo project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,12 +59,10 @@ sudo apt update
 sudo apt install libsdl2-dev
 sudo apt install build-essential
 ```
-To build the engine, editor and tests, use `cmake` and then `make`:
+To build the engine, editor and tests, use CMake from the root directory:
 ```bash
-mkdir build
```
Comment (turanszkij) [inline README.md:64]: Why do you want to complicate the build instructions, and why did you change Release to Debug?

File: README.md [hunk 2]
```diff
@@ -227,9 +225,10 @@ You can specify command line arguments (without any prefix) to switch between re
 <br/>
 
 ### Other software using Wicked Engine
-- <a href="https://www.game-guru.com/max">Game Guru MAX</a>: Easy to use game creator software
-- <a href="https://www.youtube.com/watch?v=0SxXmnSQ6Q4">Flytrap</a>: Demoscene production by qop
-- Your project: add your project to this readme and open a pull request
+- <a href="https://github.com/MolassesLover/WickedEngine-Demo.git">The Wicked Engine demo</a>: A fully-fledged game in the works.
```
Comment (turanszkij) [inline README.md:228]: This should be added later if the project is a bit more matured, and it should be added to the end of the list.

**Review by turanszkij — CHANGES_REQUESTED** [review]

**Comment by turanszkij** [comment]
I don't like the change to build instructions. The online build system uses the previous build steps, and they are simpler too. Adding the demo project to the readme can wait until it has something to show.

**Comment by MolassesLover** [comment]
You should probably add `-j $(nproc)` to the `make` command, though. It drastically speeds up compilation. It should make the GitHub CI faster and help out newer developers.
